recipe/meta.yaml

```diff
@@ -33,8 +36,10 @@ test:
 about:
     home: https://github.com/geopython/pycsw
     license: MIT
-    summary: OGC Catalog Service for the Web (CSW) implementation in python
+    summary: 'OGC Catalog Service for the Web (CSW) implementation in python.'
 
 extra:
     recipe-maintainers:
+        - kalxas
+        - tomkralidis
```

Comment:
@kalxas and @tomkralidis is that OK? Once merged you will get commit rights to this feedstock and will be able to merge PRs here.

For more details on what does being a conda-forge feedstock maintainer entail see https://github.com/conda-forge/staged-recipes#10-what-does-being-a-conda-forge-feedstock-maintainer-entail